src/components/stripe-component/index.js

```diff
@@ -25,10 +25,11 @@ const StripeProvider = ({ userProfile, reservation, payTicket, providerKey, prov
     const stripePromise = useMemo(() => loadStripe(providerKey), [providerKey])
 
     const options = {
-        fonts: stripeOptions?.fonts,
+        // fonts: stripeOptions?.fonts,
```

Comment:
why this is commented ?

Reply:
I'll update the PR adding this settings. Thank you